docs/source/_src_user_guide/state_propagation/propagation_setup/dynamics_types/mass.rst

```diff
@@ -4,93 +4,84 @@
-an :ref:`example`). Propagating the mass of a body is typically (but not exclusively) coupled with the use of a thrust model.
-See the page on :ref:`thrust_models` for more details..
-
-On this page, only the Tudat-native objects necessary as input will be described. For all the other inputs, please
-refer to the related API entry (:func:`~tudatpy.numerical_simulation.propagation_setup.propagator.mass`).
+detail in the API documentation. In this page, only the Tudat-specific aspects of the input will be briefly described.
 
-.. note::
-  Settings to define the propagation of rotational dynamics reflect those specified in :ref:`translational_dynamics`
-  and :ref:`rotational_dynamics`.
-  Besides the obvious differences, an important one is that no 'central body' and no propagator type are given.
+The mass is typically propagated numerically to account for the influence of thrust on a vehicle's mass. Unlike other
+types of dynamics, there are no alternative representations (propagators) for the mass.
 
 Inputs
-=======
+======
 
-The Tudatpy-native inputs to create the settings for a translational propagator are the following:
+In addition to the settings described :ref:`here <propagation_inputs>`, the definition of rotatitonal dynamics settings requires:
 
-- a set of mass models (created via the :func:`~tudatpy.numerical_simulation.propagation_setup.create_mass_rate_models` factory function);
-- settings to terminate the propagation (see :ref:`termination_settings`);
-- dependent variables that should be saved (see :ref:`dependent_variables`).
+- A set of mass models (created via the :func:`~tudatpy.numerical_simulation.propagation_setup.create_mass_rate_models` factory function; see below)
+- The initial conditions for the propagation (initial mass and time)
 
-.. _example:
 
-Example
-========
+Mass-rate models
+================
 
-In the example below, the body "Spacecraft" will be propagated w.r.t. body "Earth", using given mass rate models (not
-provided) and a given initial mass.
-The propagation will terminate once the ``simulation_end_epoch`` epoch is reached.
-Next to that, the propagator is asked to save the total acceleration, Keplerian state, latitude, and longitude of the
-spacecraft as
-dependent variables. The time and state will be printed on the terminal once every 24 hours (simulation time), while
-the state will be propagated through the Encke formulation.
-dependent variable. The time and rotational state will be printed on the terminal once every 24 hours (simulation
-time).
+The setup of a mass rate model in Tudat is substantially simpler than for the :ref:`accelerations <acceleration_model_setup>` and :ref:`torques <torque_model_setup>`.
+This is, in part, due to the very limited set of options for computing mass rates.
+
+Typically (but not necessarilly), a mass rate is directly related to a body's thrust. An example of this is shown below,
```

Comment:
maybe add a practical example of when they are decoupled (solar sailing)?

Reply:
Solar sailing is seen as radiation pressure, not thrust (normally). In principle, decoupling between mass rate and thrust is non-physical. I've added a few notes clarifying this